services/app/src/Format.test.tsx

```diff
@@ -8,3 +8,32 @@ describe('formatPlayPeriod', () => {
     expect(formatPlayPeriod(999, 999)).toEqual('2+ hrs');
   });
 });
+
+describe('smartTruncateSummary', () => {
+  it('chains smaller sentences before stopping', () => {
+    const DEAD_WASTELAND_SUMMARY = 'A story influenced by the awesome game Dead of Winter: Your colony is attacked. How will you respond? Actions have consequences, and consequences are far reaching. Will you survive The Dead of Winter?';
+    const DEAD_WASTELAND_EXPECTED = 'A story influenced by the awesome game Dead of Winter: Your colony is attacked. How will you respond?';
+    expect(smartTruncateSummary(DEAD_WASTELAND_SUMMARY)).toEqual(DEAD_WASTELAND_EXPECTED);
+  });
+
+  it('adds ellipses at a sentence boundary for a more natural feel', () => {
+    const SHARDS_OF_TIME_SUMMARY = 'You wake-up in the middle of an Ash Barren wasteland of Aikania. Now you have to explore the lands but dark creatures are tormenting this land. Can you free Aikania from this horrible curse?';
+    const SHARDS_OF_TIME_EXPECTED = 'You wake-up in the middle of an Ash Barren wasteland of Aikania...';
+    expect(smartTruncateSummary(SHARDS_OF_TIME_SUMMARY)).toEqual(SHARDS_OF_TIME_EXPECTED);
+  });
+
+  it('leaves excessively-long sentences alone', () => {
+    const MUNROE_SUMMARY = 'This kid-friendly, spooky Halloween adventure takes you into Mr Monroe’s haunted mansion where unexplainable things are happening…';
+    expect(smartTruncateSummary(MUNROE_SUMMARY)).toEqual(MUNROE_SUMMARY);
+  });
+});
+
+describe('pluralize', () => {
```

Comment:
Can we switch to using https://github.com/blakeembrey/pluralize, which also supports irregular pluralizations and singularizations, and has no extra dependencies / doesn't bloat the package :)

Reply:
Done.